src/e3/anod/spec.py

```diff
@@ -16,12 +16,19 @@
 from e3.anod.qualifiers_manager import QualifiersManager
 from e3.yaml import load_with_config
 
-# CURRENT API version
+# Default API version
 __version__ = "1.4"
 
-SUPPORTED_API = (__version__, "1.5")
-# The driver can support multiple version of the spec API, we currently support
-# only the version 1.4 and 1.5. Default is still 1.4
+SUPPORTED_API = (__version__, "1.5", "1.6")
+# API VERSIONS
+#
+# Version 1.4 (initial version)
+# Version 1.5
+#    NEW: YAML files are also searched in subdirectories whose basename is the
```

Comment:
If the specs repo declares version 1.5 and above, the YAML files will only be searched in subdirectories; see e3.anod.spec.load_config_file

Note that you should create a new entry in NEWS.md.

Reply:
Thanks I will review that. In loader.py files were still searched in root dir